### PR TITLE
Des 544

### PR DESCRIPTION
**Pull request by StommePoes**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -27,32 +55,106 @@ module.exports = {
...
+    }),
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV' : JSON.stringify(process.env.NODE_ENV)
+    }),
+    new webpack.NamedModulesPlugin()
+  ]
 };
+
+
+// module.exports = {
```
Comment (wyseguyonline) [inline webpack.config.js]: Can we lose these commented lines?

Comment (briandunnpearson) [reply webpack.config.js]: Definitely.  Deleted now.

**Comment by briandunnpearson** [comment]
I'm closing this PR because the implementation does not match what QA is looking for.

**Comment by briandunnpearson** [comment]
Re-opening after pushing one commit and review by Payal.

**Comment by udhadpa** [comment]
Tests written and approved. Good to merge

**Pull request merged**